Tests/pens/ttGlyphPointPen_test.py

```diff
@@ -1,24 +1,25 @@
 import os
-import unittest
+import pytest
 import struct
 
 from fontTools import ttLib
-from fontTools.misc.testTools import TestCase
-from fontTools.pens.ttGlyphPointPen import TTGlyphPointPen, MAX_F2DOT14
-from fontTools.ttLib.tables import ttProgram
+from fontTools.pens.ttGlyphPen import TTGlyphPointPen, MAX_F2DOT14
 
 
-class TTGlyphPointPenTest(TestCase):
-
+class TTGlyphPointPenTest:
```

Comment:
you should move this test class in the same test module as ttGlyphPen_test.py, since we decided to also move the class itself to the same Lib module. The test directory has the same layout as the Lib one.